README.md

```diff
@@ -1,2 +1,79 @@
-# react-with-nodejs-sample
-This sample demonstrates the dashboard rendering in your Bold BI server in React with NodeJS
+# BoldBI Embedding React with NodeJS Sample
+
+This project was created using NodeJS 14.16. This application aims to demonstrate how to render the dashboard available on your Bold BI server.
```

Comment:
This project is created using NodeJS 14.16. This application aims to demonstrate how to render the dashboard available on your Bold BI server.